sparrow_mlpipes/bins/source.py

```diff
@@ -37,7 +37,11 @@ def make_source_bin(input_uri: str, index: int = 0) -> Gst.Bin:
     uridecodebin = make_element(
         "uridecodebin", "uri-decode-bin", uri=f"file://{Path(input_uri).absolute()}"
     )
+    # multifilesrcbin = make_element(
+    #     "multifilesrc", "multi-file-source", location="%03d.jpeg", index=0
+    # )
```

Comment:
The first arg is "factory name", second arg is "element name" . . . not sure what the difference is.
Also need to think of way to handle an arbitrary directory vs. what I hard-coded into the command line test.

Reply:
The factory name is the actual name of an element like `uridecodebin` whereas the element name is any string you want to use to identify that element